enterprise/app/filter/filter.tsx

```diff
@@ -99,15 +130,50 @@ export default class FilterComponent extends React.Component<FilterProps, State>
     const startDateParam = this.props.search.get(START_DATE_PARAM_NAME);
     const endDateParam = this.props.search.get(END_DATE_PARAM_NAME);
 
-    const startDate = (startDateParam ? moment(startDateParam) : moment().subtract(7, "days")).toDate();
-    const endDate = (endDateParam ? moment(endDateParam) : moment()).toDate();
+    const defaultStartDate = getDefaultStartDateForPage(this.props.path);
```

Comment:
Let's just keep it simple and do Last 30 days for everything instead of having a different default start date per page.